src/NeptuneSchema.js

```diff
@@ -220,7 +238,8 @@ async function getEdgesProperties() {
 
 
 async function getNodeProperties(node) {
-    let query = `MATCH (n:${node.label}) RETURN properties(n) as properties LIMIT ${SAMPLE}`;        
+    let query = `MATCH (n:${node.label}) RETURN properties(n) as properties LIMIT ${SAMPLE}`;
```

Comment:
Possibility of code injection here?

Reply:
@sophiadt we will need to change the open cypher queries to be parameterized.

Querying via http can be done by having separate `query` vs `parameters` in the request:  https://docs.aws.amazon.com/neptune/latest/userguide/opencypher-parameterized-queries.html

Querying via the SDK can be done by specifying `openCypherQuery` and `parameters` in the input for `ExecuteOpenCypherQueryCommand`: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/neptunedata/command/ExecuteOpenCypherQueryCommand/

We should modify the `queryNeptune` function to accept the string query and also the query parameters. Callers of this method will need to be modified to pass the query and parameters separately.